crates/router/src/core/user.rs

```diff
@@ -335,7 +329,20 @@ pub async fn reset_password(
         .await
         .change_context(UserErrors::InternalServerError)?;
 
-    //TODO: Update User role status for invited user
+    if let Some(invited_merchant_id) = token.get_merchant_id() {
```

Comment:
we can change this variable name to something better.

Reply:
Changed to `inviter_merchant_id`.